tests/test_osm_tools/test_osmconvert.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+import unittest.mock as mock
+import os
+import cosmocrat.definitions as definitions
+
+from cosmocrat.osm_tools import osmconvert
+from tests.definitions import EMPTY_STRING, FILE_PATH, FILE_NAME, FILE_FORMAT, FAKE_MULTI_PURPOSE
+from tests.helpers.helpers import Expected_In_Arg
+
+SUBPROCESS_NAME=osmconvert.SUBPROCESS_NAME
+OSMCONVERT_MODULE_PATH='cosmocrat.osm_tools.osmconvert'
+
+@pytest.mark.osm_tools
+@mock.patch('cosmocrat.definitions.OSMCONVERT_PATH', FAKE_MULTI_PURPOSE)
+class TestOsmconvert():
+    @mock.patch(f'{OSMCONVERT_MODULE_PATH}.subprocess_get_stdout_output')
+    def test_get_osm_file_timestamp(self, mock_get_stdout):
+        input_path = FAKE_MULTI_PURPOSE
+
+        result = osmconvert.get_osm_file_timestamp(input_path)
+
+        # arguments in command
+        mock_get_stdout.assert_called_with(
+            args=[definitions.OSMCONVERT_PATH,
+            '--out-timestamp',
+            input_path,
+            mock.ANY],
+            subprocess_name=SUBPROCESS_NAME)
+
+        # result
+        assert result == mock_get_stdout.return_value
+
+    @mock.patch(f'{OSMCONVERT_MODULE_PATH}.run_command_wrapper')
+    def test_set_osm_file_timestamp(self, mock_run_command_wrapper):
+        input_path = FAKE_MULTI_PURPOSE
+        timestamp = FAKE_MULTI_PURPOSE[::-1]
+
+        with mock.patch('os.rename') as mock_rename, \
+        mock.patch(f'{OSMCONVERT_MODULE_PATH}.uuid4', return_value='uuid') as mock_uuid4, \
+        mock.patch(f'{OSMCONVERT_MODULE_PATH}.deconstruct_file_path',
```

Comment:
can we indent?

Reply:
it's a long `with` statement, no need for indentation